### PR TITLE
Booksplash styles

### PR DESCRIPTION
**Pull request by nathanstitt**
Finally figured out how to interpolate a selector in less.

This styles the following components

Desktop SVG for blank CC course
image

CC Dashboard
image

Course chooser
image

Student Dashboard:

image

### CODE REVIEW

**Comment by nathanstitt** [comment]
@Fredasaurus this sets the CC dashboard to be 120px tall with 88px font.  I left the Tutor dashboard the same with the font at 35rem on a  24rem background (it overflows).

Stacking is defined as 'Helvetica Neue', Helvetica, Arial, sans-serif, with letter spacing at -6px to pull it in a bit.

File: resources/styles/components/cc-dashboard/index.less [hunk 1]
```diff
@@ -159,50 +149,22 @@
...
+          .banner       { fill: @bg; }
+          .banner-light { fill: lighten(@bg, 10%); }
+          .course-name  { fill: @@fg; }
+        }
       }
-      &.anatomy_physiology {
-        .banner       { fill: @anatomy_physiology-bg; }
-        .banner-light { fill: lighten(@anatomy_physiology-bg, 10%); }
-        .course-name  { fill: @anatomy_physiology-fg; }
+      .for(@openstax-book-appearance-codes); .-each(@code) {
```
Comment (philschatz) [inline resources/styles/components/cc-dashboard/index.less]: Ah, you opted against `car` and `cdr` and using something like:

```less
.foo() { } // empty list case
.foo(@head; @tail...) {
   // stuff
  .foo(@tail...);
}
```

Comment (nathanstitt) [inline resources/styles/components/cc-dashboard/index.less]: Yeah, I ran across the `.for`mixin and think it expresses what's going on better.  Internally it's using that pattern of course,  as far as I know that's the only real way to loop in less.

Comment (philschatz) [inline resources/styles/components/cc-dashboard/index.less]: The `.for(@array)` implementation looks a bit odd but yeah, semantically they're (hopefully) the same :smile:

**Pull request merged**